### ISSUE TITLE
cd devos; nix-shell = error: infinite recursion encountered

### ISSUE THREAD

**Issue by milahu** [divnix/devos#394]
## Reproduce

```
git clone https://github.com/divnix/devos.git --depth 1
cd devos
nix-shell
```

## Actual

```
error: infinite recursion encountered

       at /nix/store/0r0xrqj5l88071cama7hlha5ajya06c8-source/lib/modules.nix:525:9:

          524|     in warnDeprecation opt //
          525|       { value = builtins.addErrorContext "while evaluating the option `${showOption loc}':" value;
             |         ^
          526|         inherit (res.defsFinal') highestPrio;
(use '--show-trace' to show detailed location information)
```
![meme jackie chan what why 200x200](https://user-images.githubusercontent.com/12958815/138598278-6cf06506-f6a8-40da-969c-c8710067981c.jpeg)

## Possible Solution
rollback to a088edd via

```
git clone https://github.com/divnix/devos.git --depth 1
cd devos
git pull --depth 100
git checkout main
while true
do
  git checkout HEAD~1 && nix-shell && break
done
```

the `error: infinite recursion encountered` first appears with dcb4253

```
git rev-list --count main ^a088edd
61
```

## Your Environment
```
nix --version 
nix (Nix) 2.5pre20211007_844dd90
```

**Comment by Pacman99** [comment]
This is fixed with #403, because we made a custom fork of flake-utils-plus to drop patching support. I just need to get that PR merged.

**Comment by Pacman99** [comment]
Closing since #403 is merged, if the issue still appears feel free to re-open
